packages/contracts/DKIMRegistry.sol

```diff
@@ -9,15 +9,15 @@ import "@openzeppelin/contracts/access/Ownable.sol";
  */
 contract DKIMRegistry is Ownable {
     // Mapping from domain name to DKIM public key hash
-    mapping(string => uint256) public dkimPublicKeyHashes;
+    mapping(string => bytes32) public dkimPublicKeyHashes;
 
     constructor() {
         // Set values for popular domains
-        dkimPublicKeyHashes["gmail.com"] = uint256(20579775636546222313859320423592165398188168817714003219389601176739340973605);
-        dkimPublicKeyHashes["hotmail.com"] = uint256(2750248559912404074361997670683337416910370052869160728223409986079552486582);
-        dkimPublicKeyHashes["twitter.com"] = uint256(12431732230788297063498039481224031586256793440953465069048041914965586355958);
-        dkimPublicKeyHashes["ethereum.org"] = uint256(13749471426528386843484698195116860745506750565298853141220185289842769029726);
-        dkimPublicKeyHashes["skiff.com"] = uint256(11874169184886542147081299005924838984240934585001783050565158265014763417816);
+        dkimPublicKeyHashes["gmail.com"] = bytes32(uint256(21238126716164910617487233347059218993958564577330259377744533585136010170208));
+        dkimPublicKeyHashes["hotmail.com"] = bytes32(uint256(2431254542644577945126644490189743659677343436440304264654087065353925216026));
+        dkimPublicKeyHashes["twitter.com"] = bytes32(uint256(5857406240302475676709141738935898448223932090884766940073913110146444539372));
+        dkimPublicKeyHashes["ethereum.org"] = bytes32(uint256(1064717399289379939765004128465682276424933518837235377976999291216925329691));
+        dkimPublicKeyHashes["skiff.com"] = bytes32(uint256(7901875575997183258695482461141301358756276811120772965768802311294654527542));
```

Comment:
For people to verify, I wonder if we should make this more legible in solidity. So i.e. if we have comments with the original raw key and a link to a 3 line repl.it that gets from dns and hashes it.

Reply:
True. We can add something.
But the raw key is also just is a large number array. Users would still need the selector and some method to convert to dns dkim value to 17*121 chunk